packages/components/src/ListReview/components/Row.tsx

```diff
@@ -57,6 +59,18 @@ export const RowContainer = styled.tr`
       min-width: 100px;
     }
   }
+
```

Comment:
Why is this required?
On mobile the rows stack. Can width not be 100%

Reply:
There are cases when long names, emails (which are in the second column), etc., the 100% td horizontally overflows and breaks the display (aka breaks the third column, pushing it out of the screen, breaking the content inside it), hence limiting them to 200px on max-width 576px.


![Screenshot 2025-01-21 at 1 07 07 PM](https://github.com/user-attachments/assets/75634c01-c7a1-4207-9a37-2055019baebe)